docs/guides/external-node/08_pruning.md

```diff
@@ -0,0 +1,38 @@
+# Pruning
+
+It is possible to configure Node to periodically remove all data from batches older than a threshold, both from Postgres
+and from tree.
```

Comment:
- Nit: Why is "Node" capitalized?
- IMO, "from batches" clarification is unnecessary detail; it'd be sufficient to say "data older than a configurable threshold". OTOH, it could make sense to mention that data cannot be removed until certain unchangeable conditions apply (e.g., unexecuted L1 batches and associated data cannot be removed).

Reply:
Node as in ZKsync Node, I've added ZKsync there.
Imo the best place to write about this restriction is best under configuration. I don't feel like it would be important to most Node operators.